solarforecastarbiter/metrics/probabilistic.py

```diff
@@ -224,7 +318,7 @@ def reliability(obs, fx, fx_prob):
         The reliability of the forecast [unitless], where a perfectly reliable
         forecast has value of 0.
 
-    See Also:
+    See Also
```

Comment:
good catch. remove the extra `-` from the line below too.

Reply:
Thanks for reminding me. Removing now.